src/styles/_colors.scss

```diff
@@ -20,6 +24,10 @@ $blue-300: #5291ff;
 $blue-400: #5274ff;
 
 // Green
+$green-11: #01f4cb;
+$green-12: #148071;
```

Comment:
Nice thanks for adding the colours here, we need to keep this up when colours from figma designs aren't currently listed in this file.